src/WebJobs.Script/Rpc/Capabilities.cs

```diff
@@ -14,7 +15,7 @@ internal class Capabilities
 
         public Capabilities(ILogger logger)
         {
-            _logger = logger;
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
```

Comment:
This is new surface. Any reason why this is taking an `ILogger` and not an `ILogger<T>`?

Reply:
This is specific instance of the workerLogger that is initialized here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs#L84 using worker id and the runtime. Let me know if there is a better way to use this logger in capabilities